internal/aws/albrgt/rgt.go

```diff
@@ -147,6 +147,35 @@ func (r *RGT) GetClusterResources() (*Resources, error) {
 		}
 	}
 
+	// Legacy deployments may not have the proper tags, and RGT doesn't allow you to use wildcards on names
+	p := request.Pagination{
+		EndPageOnSameToken: true,
+		NewRequest: func() (*request.Request, error) {
+			req, _ := r.GetResourcesRequest(&resourcegroupstaggingapi.GetResourcesInput{
```

Comment:
Couldn't this use `GetResourcesPages` too, like that call we changed a couple of days ago in #605?

Reply:
Yeah we can. I went with the old method since we hadn’t merged that yet.